app/build.gradle

```diff
@@ -122,6 +122,7 @@ dependencies {
 	implementation 'androidx.lifecycle:lifecycle-common-java8:2.2.0'
 	implementation 'androidx.room:room-runtime:2.2.5'
 	kapt 'androidx.room:room-compiler:2.2.5'
+	implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
```

Comment:
do we need a gradle witness line for this?

Reply:
No additional line, as [it is already there](https://github.com/grote/Transportr/blob/master/app/witness.gradle#L45) - that's the weirdest part of it. The library was already included by some other dependency (I assume that therefore the app didn't crash afterwards), but for some reason it needs the explicit dependency to avoid the first-start crash... Don't try to actually understand Android development :facepalm:

Reply:
Yeah it is full of endless mysteries... I guess you've verified that this really fixes things on a fresh install.

Reply:
Yes, I did, at least on three or four different emulators.